cpp/log/cluster_state_controller-inl.h

```diff
@@ -0,0 +1,203 @@
+#ifndef CERT_TRANS_LOG_CLUSTER_STATE_CONTROLLER_INL_H_
+#define CERT_TRANS_LOG_CLUSTER_STATE_CONTROLLER_INL_H_
+
+#include "log/cluster_state_controller.h"
+#include "log/etcd_consistent_store-inl.h"
```

Comment:
The ```-inl``` one? I'd also put a blank line between the ```cluster_state_controller.h``` one and the rest, it happens to be at the top, but a ```-inl.h``` is sort of morally equivalent to a ```.cc``` file, I'd say...